src/status_im2/contexts/chat/lightbox/bottom_view.cljs

```diff
@@ -49,14 +49,14 @@
 
 (defn bottom-view
   [messages index scroll-index insets animations derived item-width props]
-  (let [text               (get-in (first messages) [:content :text])
+  (let [_text              (get-in (first messages) [:content :text])
         padding-horizontal (- (/ item-width 2) (/ c/focused-image-size 2))]
     [reanimated/linear-gradient
      {:colors [:black :transparent]
       :start  {:x 0 :y 1}
       :end    {:x 0 :y 0}
       :style  (style/gradient-container insets animations derived)}
-     [rn/text {:style style/text-style} text]
+     #_[rn/text {:style style/text-style} text]
```

Comment:
there should also be an issue to reenable this feature. 

Can you leave a TODO with that issue? If the issue does not exist you can go ahead and create it 👍